Datadog/TargetSupport/DatadogCrashReportingTests/DatadogCrashReportingTests.xcconfig

```diff
@@ -15,5 +11,11 @@
 //       One of the two will be used. Which one is undefined.
 //       """
 //
-//       Because some mocks in 'DatadogCrashReportingTests' use PLCR type's subclassing, their runtime cast with 'as? <PLCR_TYPE>' fails
-//       resulting with tests failures. Until a workaround or a better solution is found, this instrumentation must be disabled.
+//       Because some mocks in 'DatadogCrashReportingTests' use PLCR type's subclassing, their runtime cast with 'as? <PLCR_TYPE>' could fail depending in which library is found first
+// We force 'DatadogCrashReporting' to be linked before linking 'DatadogSDKTesting' so PLCR symbols are found in that library.
+OTHER_LDFLAGS= $(inherited) -framework DatadogCrashReporting
+
+// Add DatadogSDKTesting instrumentation (if available in current environment)
+ #include "../xcconfigs/DatadogSDKTesting.local.xcconfig"
```

Comment:
we can have `#include?` here if `DatadogSDKTesting.local.xcconfig` is not essential

Reply:
But it is essential, right? I mean, including it is the only reason for the existence of this file (DatadogCrashReportingTests.xcconfig), it not Datadog.xcconfig could be used instead

Reply:
we have it optional in [other places](https://github.com/DataDog/dd-sdk-ios/blob/master/Datadog/TargetSupport/DatadogTests/DatadogTests.xcconfig#L5), but i don't know what is different with this file.

if everything can still work without `local.xcconfig`, let's make it optional. does that make sense?

Reply:
Ok, I will change